publisher/docker/Dockerfile

```diff
@@ -29,4 +29,4 @@ RUN apk add --update --no-cache \
 WORKDIR /home/app
 COPY --from=build /home/src/dist/swapscope .
 
-CMD ["sh", "-c", "./swapscope --nats ${NATS_HOSTS} --nats-jwt ${NATS_JWT} --nats-nkey ${NATS_NKEY} --pub-subject-prefix ${PUB_SUBJECT_PREFIX}"]
\ No newline at end of file
+CMD ["sh", "-c", "./swapscope --nats ${NATS_URL} --nats-sub-jwt ${NATS_SUB_JWT} --nats-sub-nkey ${NATS_SUB_NKEY}"]
```

Comment:
I believe `NATS_PUB_JWT` and `NATS_PUB_NKEY` are missing.

Reply:
Should we include all required variables here? That would be NATS_PUB_* and all database connection information. Currently I just fixed flags because earlier ones did not exist in the code.

Reply:
Dockerfile adjusted by DevOps: https://github.com/SyntropyNet/swapscope/pull/17

Reply:
In that case do not modify this file in order not to create merge conflicts.

Reply:
Reverted